### PR TITLE
Further trimming of the index

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
This removes some detail that really isn't needed today.

**Comment by david-a-wheeler** [comment]
@jkingdon - thoughts?

File: index.html [hunk 1]
```diff
@@ -985,6 +985,7 @@
      installation.
      </LI>
 
+<!--
```
Comment (jkingdon) [inline index.html:988]: So this gets rid of:

1. The suggestion of just installing metamath.exe and set.mm for windows. Looks fine to me. It doesn't seem like an especially important technique to highlight.
2. Remove instructions for how to get gcc for MacOS. This is a good change because the instructions given are completely and utterly out of date.
3. Remove instructions for rlwrap. Although I do consider rlwrap to be an essential part of an installation (and I'd maybe reword the parts of the README.TXT which call it optional), I guess I'm OK with pointing to the README.TXT for the details.

File: index.html [hunk 2]
```diff
@@ -1501,6 +1503,7 @@
 
 </UL>
 
+<!-- Modern tools don't need this kind of help. Thank goodness.
```
Comment (jkingdon) [inline index.html:1506]: Trying to document tools not even knowing what people are using was always a bit tricky, but what we have here is out of date enough that getting rid of it seems like an improvement.

**Review by jkingdon — APPROVED** [review]
